blog/2025-06-30-adaptive-tasklist-scaler.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Adaptive Tasklist Scaler"
+subtitle: test
+date: 2025-06-30
+authors: shaddoll
+tags:
+  - deep-dive
+  - cadence-operations
+  - cadence-matching
+---
+
+At Uber, we previously relied on a dynamic configuration service to manually control the number of partitions for scalable tasklists. This configuration approach introduced several operational challenges:
+
+- **Error-prone:** Manual updates and deployments were required.
+- **Unresponsive:** Adjustments were typically reactive, often triggered by customer reports or observed backlogs.
+- **Irreversible:** Once increased, the number of partitions was rarely decreased due to the complexity of the two-phase process, especially when anticipating future traffic spikes.
```

Comment:
it's not clear to reader what `two-phase process` is. It would be great to add a section `How did manual scale work?` (before `How Does Adaptive Tasklist Scaler Work?` section) and talk about the previous process